### ISSUE TITLE
Copy failures from `accessibility-devops` project

### ISSUE THREAD

**Issue by nfreear** [nfreear/accessibility-ci-demo#4]
..

* [@nfreear/accessibility-devops - `pages/fail.html`](https://github.com/nfreear/accessibility-devops/blob/main/pages/fail.html)
* [@nfreear/accessibility-devops - `pages/fail.css`](https://github.com/nfreear/accessibility-devops/blob/main/pages/fail.css)

..
